app/src/main/res/values/strings.xml

```diff
@@ -895,7 +895,7 @@
     <string name="suggested_edits_image_recommendations_task_get_started">Start</string>
     <string name="suggested_edits_image_recommendations_task_goal_progress">Great, keep going! 👍</string>
     <string name="suggested_edits_image_recommendations_task_goal_complete">Daily goal achieved! 👏</string>
-    <string name="suggested_edits_image_recommendations_task_goal_surpassed">You\'ve reached the goal</string>
+    <string name="suggested_edits_image_recommendations_task_goal_surpassed">You reached the goal</string>
```

Comment:
@cooltey I think the string is "You reached the daily goal"

Reply:
Done.